analytics-webapps/src/main/resources/locale/portlet/Analytics_fr.properties

```diff
@@ -234,6 +234,7 @@ analytics.connectionsCount=Nombre de connexions
 analytics.definePercentageThreshold=Calculer le pourcentage avec un seuil
 analytics.previousPeriodValue=P\u00E9riode pr\u00E9c\u00E9dente : {0}
 analytics.loadMore=Charger plus
+analytics.distinctLogins=Connexions distinctes
```

Comment:
not needed, crowdin will do it

Reply:
i think that to accelerate the test , no need to waiting the crowdin build WDYT